x-pack/test/functional/apps/upgrade_assistant/upgrade_assistant.ts

```diff
@@ -17,10 +17,11 @@ export default function upgradeAssistantFunctionalTests({
   const security = getService('security');
   const log = getService('log');
   const retry = getService('retry');
+  const testSubjects = getService('testSubjects');
 
-  // Failing: See https://github.com/elastic/kibana/issues/86546
-  describe.skip('Upgrade Checkup', function () {
-    this.tags('includeFirefox');
+  // Updated for the hiding of the UA UI.
+  describe('Upgrade Checkup', function () {
+    this.tags('skipFirefox');
```

Comment:
Is there a particular reason why we're skipping Firefox now?

Reply:
We're having some perms issues with firefox right now. Skipping FF since the functionality isn't different and going to circle back once we figure out what the issue is on FF. That will be a separate PR.